frontend/src/framework/internal/components/PersistenceManagementDialog/sessionManagementContent.tsx

```diff
@@ -0,0 +1,419 @@
+                        endAdornment={
+                            <DenseIconButton onClick={handleClearTitleFilter} title="Clear filter">
+                                <Close fontSize="inherit" />
+                            </DenseIconButton>
+                        }
+                        value={tableFilter.title ?? ""}
+                        placeholder="Search title"
+                        onChange={handleTitleFilterValueChange}
+                        className="h-6"
+                    />
+                </Label>
+                <Label text="Updated at" wrapperClassName="min-w-2xs">
+                    <DateRangePicker
+                        onChange={handleDateFilterRangeChange}
+                        className="webviz-eds-date-range-picker --compact rounded focus-within:outline-0 border border-gray-300 h-10"
+                    />
+                </Label>
+            </div>
+            <div className="flex gap-2 mb-2">
+                <Tooltip title="Start and open new session" placement="bottom" enterDelay="medium">
+                    <Button color="primary" onClick={handleNewSessionClick} variant="contained" size="medium">
+                        <Add fontSize="inherit" /> New session
+                    </Button>
+                </Tooltip>
+                <span className="grow" />
+                <Tooltip title="Edit the selected session" placement="bottom" enterDelay="medium">
+                    <Button color="primary" disabled={!selectedSessionId} onClick={handleEditClick} size="medium">
+                        <Edit fontSize="inherit" /> Edit
+                    </Button>
+                </Tooltip>
+                <Tooltip title="Open the selected session" placement="bottom" enterDelay="medium">
+                    <Button
+                        color="primary"
+                        disabled={!selectedSessionId}
+                        onClick={handleOpenSessionClick}
+                        size="medium"
+                    >
+                        <FileOpen fontSize="inherit" /> Open
+                    </Button>
+                </Tooltip>
+                <Tooltip title="Delete the selected session" placement="bottom" enterDelay="medium">
+                    <Button
+                        color="danger"
+                        disabled={!selectedSessionId || deletePending}
+                        onClick={handleDeleteClick}
+                        size="medium"
+                    >
+                        {deletePending ? <CircularProgress size="small" /> : <Delete fontSize="inherit" />} Delete
+                    </Button>
+                </Tooltip>
+                <Tooltip title="Refresh list" placement="top" enterDelay="medium">
+                    <Button color="primary" onClick={handleRefreshClick} size="medium">
+                        {sessionsQuery.isFetching ? <CircularProgress size="small" /> : <Refresh fontSize="inherit" />}{" "}
```

Comment:
Consider using `isRefetching` flag, thus we do not animate on every fetch, only when actively re-fetching.